src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityEnum.cs

```diff
@@ -605,6 +605,169 @@ public AccessibilityStates(params AccessibilityState[] states)
+        ColumnInserted          = 12,
+        /// <summary>
+        /// Column reordered event.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        ColumnReordered         = 13,
+        /// <summary>
+        /// Column deleted event.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        ColumnDeleted           = 14,
+        /// <summary>
+        /// Text bounds changed event.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        TextBoundsChanged       = 15,
+        /// <summary>
+        /// Text selection changed event.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        TextSelectionChanged    = 16,
+        /// <summary>
+        /// Text changed event.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        TextChanged             = 17,
+        /// <summary>
+        /// Text attributes changed event.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        TextAttributesChanged   = 18,
+        /// <summary>
+        /// Text caret moved event.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        TextCaretMoved          = 19,
+        /// <summary>
+        /// Attributes changed event.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        AttributesChanged       = 20,
+        /// <summary>
+        /// Moved out event.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        MovedOut                = 21,
+    };
+
+    /// <summary>
+    /// A collection of AccessibilityObjectEvent to be suppressed
+    /// </summary>
+    /// <seealso cref="AccessibilityObjectEvent"/>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class AccessibilitySuppressedObjectEvents
+    {
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public AccessibilitySuppressedObjectEvents(params AccessibilityObjectEvent[] objectEvents)
+        {
+            foreach (var objectEvent in objectEvents)
+            {
+                BitMask |= (1UL << (int)objectEvent);
+            }
+        }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        internal ulong BitMask { get; set; } = 0UL;
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool this[AccessibilityObjectEvent objectEvent]
+        {
+            get
+            {
+                return Convert.ToBoolean(BitMask & (1UL << (int)objectEvent));
+            }
+            set
+            {
+                if (value)
+                {
+                    // Set N-th bit
+                    BitMask |= (1UL << (int)objectEvent);
+                }
+                else
+                {
+                    // Clear N-th bit
+                    BitMask &= ~(1UL << (int)objectEvent);
+                }
```

Comment:
[Continuation of my comment about `Owner`]:
If using such a solution, we would also need to call `Interop.SetSuppressedEvents(Owner.SwigCPtr, BitMask)` here, at the end of `set`, so that changes are propagated back to DALi. (Of course, there would need to be an `internal View Owner { get; set; }` field in this structure.)

Reply:
It that valid Owner.SwigCPtr... calling "set" before "get" ? 
Owner would be not set yet.
there was not a crash... it seems that it is valid... but I do not get it how it works yet.